README.md

```diff
@@ -100,6 +101,26 @@ General security best practices apply:
   * Ultimately, the security of a system depends on the capabilities of its administrator.
   * Care should be taken when installing new software; only install from official sources that the developers indicate on their official website/github/etc.
 
+# Threat Modeling
+
+The first and most important step for security and privacy is to create a threat model. After all, how can we have security when we don't even know what we're securing against? Each person will have their own needs so everyone's threat model will be different. Threat models tend to evolve over time as our situation changes, so be sure to periodically reassess your threat model.
```

Comment:
Perhaps for the sake of this section we can simply delineate security and privacy practice into two domains: hygiene and threat modeling. The former is something everyone should do regardless (use unique passwords, multifactor authentication, disk encryption, etc) and should not really require much thought. The latter is more deliberate and requires significant effort, and discipline. This distinction is important - what do you think?

Reply:
I'm not sure, I think even for basic things like MFA and disk encryption it's still useful to have people think about the adversaries they counter, that way they understand *why* they should use it and are less likely to avoid it due to inconvenience.

Reply:
Let's definitely include them as mitigations, but also how about something to the effect of, security hygiene practices cover 90% of likely adversaries and threat modeling helps close the gap on the remaining 10%, which is significantly more difficult?

Reply:
Can you give an example of how you'd want that set up? Maybe you can edit the PR and at least get the bones of how you want it structured

Reply:
No worries, I can make a follow-up PR. Let's just lay the foundation for now.